src/test/java/org/reactome/release/cosmicupdate/COSMICUpdateUtilTest.java

```diff
@@ -0,0 +1,236 @@
+package org.reactome.release.cosmicupdate;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.SchemaClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class COSMICUpdateUtilTest
+{
+
+	private static final String IDENTIFIER_COSF1234 = "COSF1234";
+	private static final String IDENTIFIER_1234 = "1234";
+	private static final long DBID_1235 = 1235L;
+	private static final long DBID_1234 = 1234L;
+	private static final String IDENTIFIER_COSM1111 = "COSM1111";
+	private static final String IDENTIFIER_44444 = "44444";
+	private static final String IDENTIFIER_5678 = "5678";
+	private static final String IDENTIFIER_COSF6321 = "COSF6321";
+	private static final String IDENTIFIER_COSF7890 = "COSF7890";
```

Comment:
Thanks for adding constants for these!  Instead of the dummy value in the name, could the identifier's purpose be stated.  What is the difference between the usage of the identifiers "44444" and "5678", for example?

Reply:
They're just random test numbers, no deeper purpose. By putting the value into the name, I know exactly what it is without having to refer back to the variable declaration.

Reply:
Okay.  I won't push it, but if they don't have meaning you could consider naming them something like "DUMMY_IDENTIFIER_1", "DUMMY_IDENTIFIER_2", "DUMMY_DB_ID_1", "DUMMY_COSM_1", etc. so it's clear the values assigned just need to be distinct and don't have a meaning.

I'll leave that to your discretion, though.